README.md

```diff
@@ -1,5 +1,7 @@
 # FairPass
 
+[![Build](https://github.com/gitcoindev/FairPass/workflows/Build/badge.svg)](https://github.com/gitcoindev/FairPass/actions?query=workflow%3ABuild) [![Test](https://github.com/gitcoindev/FairPass/workflows/Test/badge.svg)](https://github.com/gitcoindev/FairPass/actions?query=workflow%3ATest) [![Static code analysis](https://github.com/gitcoindev/FairPass/workflows/Static%20code%20analysis/badge.svg)](https://github.com/gitcoindev/FairPass/actions?query=workflow%3AStatic%20code%20analysis)
```

Comment:
I noticed the badges point to my fork, after merging the github.com/gitcoindev path will change to github.com/fairDataSociety, apart from that no changes will be needed.